Source/Scene/Cesium3DTileset.js

```diff
@@ -1246,6 +1332,101 @@ define([
+                        tileset._hasMixedContent = true;
+                    }
+                    tile = tile.parent;
+                }
+            }
+
+            if (defined(tile)) {
+                if (!tile.selected) {
+                    tile._finalResolution = (tile === original || tile.refine === Cesium3DTileRefine.ADD);
+                    tile.selected = true;
+                    tile._selectedFrame = frameNumber;
+                    selectionQueue.push(tile);
+                }
+            } else {
+                // if no ancestors are ready, traverse down and select ready tiles to minimize empty regions
+                descendantStack.push(original);
+                while (descendantStack.length > 0) {
+                    tile = descendantStack.pop();
+                    var children = tile.children;
+                    var childrenLength = children.length;
+                    for (var j = 0; j < childrenLength; ++j) {
+                        var child = children[j];
+                        if (child.contentReady) {
+                            if (!child.selected) {
+                                child._finalResolution = true;
+                                child.selected = true;
+                                child._selectedFrame = frameNumber;
+                                touch(tileset, child, outOfCore);
+                                selectionQueue.push(child);
+                            }
+                        }
+                        if (child._depth - original._depth < 2) { // prevent traversing too far
+                            if (!child.contentReady || child.refine === Cesium3DTileRefine.ADD) {
+                                descendantStack.push(child);
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        selectionQueue.trim();
+    }
+
+    function markTilesAsFinal(tiles) {
+        var length = tiles.length;
+        tiles = tiles.internalData;
+        var i;
+        for (i = 0; i < length; ++i) {
+            tiles[i]._finalResolution = true;
+        }
+
+        for (i = 0; i < length; ++i) {
+            var parent = tiles[i].parent;
+            while (defined(parent)) {
+                // tiles using additive refinement are always final
+                parent._finalResolution = (parent.refine === Cesium3DTileRefine.ADD);
+                parent = parent.parent;
+            }
+        }
+    }
+
+    var selectionState = {
+        processingQueue: new ManagedArray(),
+        nextQueue: new ManagedArray(),
+        finalQueue: new ManagedArray(),
+        selectionQueue: new ManagedArray(),
+        done: false
+    };
+
+    function sortForLoad(a, b) {
+        var diff = b._sse - a._sse;
+        if (diff === 0 || a.refine === Cesium3DTileRefine.ADD || b.refine === Cesium3DTileRefine.ADD) {
+            return a.distanceToCamera - b.distanceToCamera;
+        } else {
```

Comment:
Just `return diff;`.  Remove the `else` clause since this is the end of the function.